docs/guidelines/branch-naming.md

```diff
@@ -0,0 +1,25 @@
+# Branch Naming Best Practices
+
+### TL;DR
+- Keep it short and concise, but make sure to include relevant key words.
+- Use category words to easily identify the type of the task.
+- Include ID of related issues to help tracking of progress.
+- Adding the name of the author helps to keep track of shared work.
```

Comment:
So the author name should be included in the branch name? Not really sure if this is good idea, since there might be multiple people working on one branch.